### PR TITLE
Bcf 3281 ability to deploy node using docker compose

### PR DESCRIPTION
**Pull request by Atrax1**
Ticket : [BCF-3281](https://smartcontract-it.atlassian.net/browse/BCF-3281)

### Resolves Dependencies

[BCF-3281]: https://smartcontract-it.atlassian.net/browse/BCF-3281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tools/docker/compose [hunk 1]
```diff
@@ -12,32 +11,26 @@ else
...
   echo "compose -- A helper script for running common docker-compose commands\
 
 Commands:
     help                  Displays this help menu
     clean                 Remove any containers and volumes related to compose files
     logs                  Display the logs of any service(s) by name
-
-    cld                   Runs the chainlink node container in dev mode
-
-    acceptance            Run the services required to perform an acceptance test
```
Comment (pavel-raykov) [inline tools/docker/compose:33]: why did you decide to remove this functionality?

Comment (Atrax1) [reply tools/docker/compose:33]: I haven't its just renaming as it was just a docker compose up behind and not running any acceptance test

Comment (pavel-raykov) [reply tools/docker/compose:33]: I see, that was a bad naming then.

Comment (pavel-raykov) [reply tools/docker/compose:33]: Could you please update the README file accordingly then?

File: tools/docker/docker-compose.yaml [hunk 2]
```diff
@@ -32,8 +30,6 @@ services:
       dockerfile: core/chainlink.Dockerfile
     entrypoint: /bin/sh -c "chainlink -c /run/secrets/config node start -d -p /run/secrets/node_password -a /run/secrets/apicredentials"
     restart: always
-    environment:
```
Comment (pavel-raykov) [inline tools/docker/docker-compose.yaml:35]: Sorry, could you confirm that there is no application that runs "docker-compose -f docker-compose.yaml ..." which assumes that CL_DATABASE_URL is propagated somehow from the upstream?

Comment (Atrax1) [reply tools/docker/docker-compose.yaml:35]: you mean if used without the compose script and hence without docker-compose.postgres.yaml ?

Comment (pavel-raykov) [reply tools/docker/docker-compose.yaml:35]: yes

File: tools/docker/compose (hunk 1 above)
Comment (Atrax1) [reply tools/docker/compose:33]: Yes I'll be pushing soon the updates on the README, PR is still in draft

File: tools/docker/docker-compose.yaml (hunk 2 above)
Comment (Atrax1) [reply tools/docker/docker-compose.yaml:35]: Its not meant to be used without compose script

File: tools/docker/README.md [hunk 3]
```diff
@@ -1,159 +1,79 @@
...
-###
+## Dev
 
-### Doing local development on the core node
-
-Doing quick, iterative changes on the core codebase can still be achieved within the compose setup with the `cld` or `cldo` commands.
-The `cld` command will bring up the services that a chainlink node needs to connect to (parity/geth, postgres), and then attach the users terminal to a docker container containing the host's chainlink repository bind-mounted inside the container at `/usr/local/src/chainlink`. What this means is that any changes made within the host's repository will be synchronized to the container, and vice versa for changes made within the container at `/usr/local/src/chainlink`.
-
-This enables a user to make quick changes on either the container or the host, run `cldev` within the attached container, check the new behaviour of the re-built node, and repeat this process until the desired results are achieved.
+Will run one node with a postgres database and optionally by default a devnet RPC node that can be either geth or parity.
```
Comment (pavel-raykov) [inline tools/docker/README.md]: does this "optionally" make sense? (I.e., it just runs one node with postgres db and a devnet RPC [parity by default]).

Comment (pavel-raykov) [reply tools/docker/README.md]: similar for "./compose up"

File: tools/docker/README.md [hunk 4]
```diff
@@ -1,159 +1,79 @@
...
+The docker-compose configuration present in this directory allows to quickly run a local development environment.
 
 # Requirements
 
 - [docker-compose](https://docs.docker.com/compose/install/)
 
 # Using the compose script
 
-Inside the `chainlink/tools/docker` directory, there is a helper script that is included which should cover all cases of integration / acceptance / development needs acroos multiple services. To see a list of available commands, perform the following:
+Inside the `chainlink/tools/docker` directory, there is a helper script that is included
```
Comment (pavel-raykov) [inline tools/docker/README.md]: is this still a part of the sentence?

File: tools/docker/README.md [hunk 5]
```diff
@@ -1,159 +1,79 @@
...
+To see a list of available commands, perform the following:
 
 ```sh
 cd tools/docker
 ./compose help
 ```
 
-## Examples
+## Env vars
```
Comment (pavel-raykov) [inline tools/docker/README.md:19]: Could you please also explain what ".env" file does in this section?

File: tools/docker/README.md (hunk 3 above)
Comment (Atrax1) [reply tools/docker/README.md]: sure will commit the changes soon 👍

File: tools/docker/README.md (hunk 4 above)
Comment (Atrax1) [reply tools/docker/README.md]: I'm updating the sentence to be more clear

File: tools/docker/README.md (hunk 5 above)
Comment (Atrax1) [reply tools/docker/README.md:19]: sure

File: tools/docker/README.md [hunk 6]
```diff
@@ -1,159 +1,85 @@
...
+The docker-compose configuration present in this directory allows to quickly run a local development environment.
 
 # Requirements
 
 - [docker-compose](https://docs.docker.com/compose/install/)
 
 # Using the compose script
 
-Inside the `chainlink/tools/docker` directory, there is a helper script that is included which should cover all cases of integration / acceptance / development needs acroos multiple services. To see a list of available commands, perform the following:
+You should use the script `compose` located in the `chainlink/tools/docker` directory
```
Comment (pavel-raykov) [inline tools/docker/README.md]: Add a dot to separate two sentences?

**Review by pavel-raykov — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**